app/src/main/java/io/multy/ui/fragments/BaseChooserFragment.java

```diff
@@ -24,12 +24,12 @@
  * Created by anschutz1927@gmail.com on 05.03.18.
  */
 
-public class BaseChooseFragment extends BaseFragment {
+public class BaseChooserFragment extends BaseFragment {
 
     @BindView(R.id.recycler_available)
-    RecyclerView recyclerAvailable;
+    RecyclerView availableRecyclerView;
```

Comment:
name of component should be first -> recyclerViewAvailableChains or similar

Reply:
it's not only for chains fragment. it's base fragment.